### PR TITLE
Add examples to dbt jinja function model docs

### PR DESCRIPTION
**Pull request by jamesowers-roo**
## Description & motivation
It was difficult to understand what I could access via the model context variable - following the docs to `graph` didn't help very much. I've added examples to help the reader find out for themselves using `log`.

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by jamesowers-roo** [comment]
🙏 Please may I have a review on this?

**Comment by mirnawong1** [comment]
hey @jamesowers-roo thanks so much for opening this PR and following up! I'll review this for you and get back to you as soon as i can ✨

File: website/docs/reference/dbt-jinja-functions/model.md [hunk 1]
```diff
@@ -6,3 +6,17 @@ id: "model"
...
 - Access the path to the model
+
+For example:
+```jinja
+{% if model.config.materialization = 'view' %}
+  {{ log(model.name ~ " is a view.", info=True) }}
+{% endif %}
+```
+
+To see the contents of `model` for a given model, you can use [log()](/reference/dbt-jinja-functions/log)
```
Comment (mirnawong1) [inline website/docs/reference/dbt-jinja-functions/model.md]:
```suggestion
To view the contents of `model` for a given model, use [log()](/reference/dbt-jinja-functions/log)
```

**Review by mirnawong1 — DISMISSED** [review]
hey @jamesowers-roo - thanks again for your patience. one tiny tweak for you to review :D

**Comment by mirnawong1** [comment]
hey @jamesowers-roo - i've approved and merged this for you. thank you again for opening this up, great flag ✨

**Review by mirnawong1 — APPROVED** [review]

**Comment by jamesowers-roo** [comment]
Apologies I missed the update again! Thank you for the edits and merge!

**Pull request merged**